src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11306,6 +11187,131 @@ static bool containsTernaryCollectionToReinterpret(ExpressionSyntax expression)
             }
         }
 
+        private (SyntaxKind operatorTokenKind, SyntaxKind operatorExpressionKind) GetOperatorTokenAndExpressionKind()
```

Comment:
have renamed to PascalCase but otherwise unchanged to keep things minimal.

Reply:
GetExpressionOperatorTokenAndKind.